torch_xla/__init__.py

```diff
@@ -1,10 +1,22 @@
 import logging
 import os
 import re
+import sys
 import tempfile
 import warnings
 
 import torch
+
+if not torch.cuda.is_available():
+  # Load _XLAC_cuda_functions to RTLD_GLOBAL, so that it can be used by _XLAC.
+  flags = sys.getdlopenflags()
+  sys.setdlopenflags(flags | os.RTLD_NOW | os.RTLD_GLOBAL)
```

Comment:
why use `os.RTLD_NOW` to perform all necessary relocations when dlopen is called, as I don't see in isuruf's example?

Reply:
Internally we discussed that it would be better, just to be safe. That's because `dlopen` needs one of `RTLD_NOW` or `RTLD_LAZY`.